<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_yazitipi">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>771</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Rapor Ekranı Yazı Tipi Biçimleri</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Yazı Tipi:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Yazı Tipi Büyüklüğü:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Renk:</string>
    </property>
   </widget>
   <widget class="QFontComboBox" name="fontComboBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>259</width>
      <height>26</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_renksecim">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>261</width>
      <height>26</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>green</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>red</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>gray</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>magenta</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>yellow</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="spinBox_yaziboyut">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>261</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="singleStep">
     <number>2</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_bold">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>50</y>
      <width>211</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Bold</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_italik">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>80</y>
      <width>211</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>İtalik</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_altcizili">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>110</y>
      <width>211</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Altı Çizili</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_calisanbilgi">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>361</width>
     <height>391</height>
    </rect>
   </property>
   <property name="title">
    <string>Çalışan Bilgileri</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ad:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Soyad:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Doğum Tarihi:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Cinsiyet</string>
    </property>
    <widget class="QRadioButton" name="radioButton_kadin">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>110</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Kadın</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton__erkek">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>110</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Erkek</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Medeni Durum</string>
    </property>
    <widget class="QRadioButton" name="radioButton_evli">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>110</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Evli</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_bekar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>110</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Bekar</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Çocuk Sayısı:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_kangrubu">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>211</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Kan Grubumu Belirteceğim</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_kangrubu">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>A RH+</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>A RH-</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>B RH+</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>B RH-</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>0 RH+</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>0 RH-</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AB RH+</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AB RH-</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="spinBox_cocuksayisi">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>270</y>
      <width>42</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="ad">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="soyad">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>110</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_baslat">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>350</y>
      <width>191</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Maaş Hesapla ve Raporla</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_raporekrani">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>250</y>
     <width>401</width>
     <height>391</height>
    </rect>
   </property>
   <property name="title">
    <string>Rapor Ekranı</string>
   </property>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>351</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
